### PR TITLE
Zero landing page [zero-222]

### PR DESCRIPTION
**Pull request by dtoki**
- adds an approval step.

- fixes issue where a prod builds are deployed to a staging environment resulting in a mismatch in FE bundled assets attempting to be loaded.

- update dist id CloudFront query to target the appropriate S3 target.

related to commitdev/zero#222

### CODE REVIEW

**Comment by dtoki** [comment]
cc @neilpower I was able to reproduce the error you were running into on the frontend app. this should fix it!

File: templates/.circleci/config.yml [hunk 1]
```diff
@@ -220,12 +221,23 @@ workflows:
             #   branches:
             #     only:  # only branches matching the below regex filters will run
             #       - /^master$/
+        
+        - wait_for_approval:
+            type: approval
+            requires:
+              - deploy_staging
 
+        - queue/block_workflow:
```
Comment (bmonkman) [inline templates/.circleci/config.yml:230]: I think this requires a circleci api key to be added as an environment variable, so we would need to do this in the project makefile.

Comment (dtoki) [reply templates/.circleci/config.yml:230]: I think we already do:

https://github.com/commitdev/zero-deployable-react-frontend/blob/dbe300be4341be431a2bfb6ae380f7a3cf2426f2/Makefile#L3

https://github.com/commitdev/zero-deployable-react-frontend/blob/dbe300be4341be431a2bfb6ae380f7a3cf2426f2/zero-module.yml#L18

Comment (dtoki) [reply templates/.circleci/config.yml:230]: right nvm! I see what you mean.

`curl -X POST --header "Content-Type: application/json" -d '{"name":"CIRCLECI_API_KEY", "value":"${CIRCLECI_API_KEY}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_API_KEY}`

**Comment by dtoki** [comment]
we should probably enforce validation during `zero init` that staging & prod root domain names can't be the same?

**Pull request merged**